charts/iot-agent-manager/values.yaml

```diff
@@ -37,7 +37,7 @@ mongo:
   # The name of the replicaset iot-agent will attempt to connect to
   rs: rs
   # The hostname of the mongodb service
-  dbhost: mongo-rs-mongodb-replicaset-0.mongo-rs-mongodb-replicaset.prod,mongo-rs-mongodb-replicaset-1.mongo-rs-mongodb-replicaset.prod,mongo-rs-mongodb-replicaset-2.mongo-rs-mongodb-replicaset.prod
+  dbhost: mongo-rs-mongodb-replicaset.prod
```

Comment:
@c0c0n3 Unfortunately i'm not using IOTA and IOTAM anymore and this PR is quite old, so i do not remember completely. But there is something in the back of my mind that this change was not as flawless as it should be and we ran into some problems with it. Could you please doublecheck that using the servicename instead of the members seperately is supported and works fine with mongo pod restarts?